docs/source/tutorial.rst

```diff
@@ -126,6 +126,113 @@ picoseconds, nanoseconds, microseconds, milliseconds, seconds, minutes, hours,
 days, years, kiloyears, megayears, gigayears, terayears and petayears,
 respectively. In the above case we decayed for one billion years.
 
+The ``calculate_decay_data()`` method can be used if, rather than just the values
+at the end of the time period, access to finer resolution decay data is required. The
+method runs ``decay()``, storing the data at each iteration and returning the
+complete data set as a pandas dataframe. The only required argument is the decay
+time period as a number, or as a numpy array with the individual decay times data
+is needed for. Optional arguments are the decay time units in the same format as for
+the ``decay()`` method, the decay units, whether the time scale should be linear or
+logarithmic and, if the decay time is given as a float, how many decay points should be
+calculated.
+
+Using ``calculate_decay_data()`` to interogate how the mass fraction of 14C decays over
+20,000 years with 14N taking it's place. The default value for the number of point to calculate
```

Comment:
You can use the following syntax if you'd prefer to have the 14's superscript:
```
 :sup:`14`\C 
 :sup:`14`\N
```
like [here](https://github.com/radioactivedecay/radioactivedecay/blob/main/docs/source/overview.rst?plain=1#L54)